src/components/CreateRecipe.js

```diff
@@ -0,0 +1,343 @@
+                  return (
+                    <Grid container item xs={12} spacing={2} key={idx}>
+                      <Grid item xs={2}>
+                        <TextField
+                          type="number"
+                          label="Amount"
+                          name={amountId}
+                          id={amountId}
+                          alt="amount"
+                          value={this.props.ingredients[idx].amount}
+                          onChange={() => {}}
+                          inputProps={{
+                            "data-id": idx,
+                            "data-field-type": "amount"
+                          }}
+                        />
+                      </Grid>
+                      <Grid item xs={2}>
+                        <InputLabel
+                          style={{ fontSize: "12px", paddingTop: "2px" }}
+                          htmlFor="age-native-simple"
+                        >
+                          Unit
+                        </InputLabel>
+                        <Select
+                          native
+                          value={this.props.ingredients[idx].unit}
+                          name={unitId}
+                          id={unitId}
+                          style={{ marginTop: "-6.1px" }}
+                          alt="unit"
+                          label="Unit"
+                          inputProps={{
+                            "data-id": idx,
+                            "data-field-type": "unit"
+                          }}
+                        >
+                          <option value="" />
+                          <option value={"piece"}>piece</option>
+                          <option value={"ml"}>ml</option>
+                          <option value={"g"}>g</option>
+                          <option value={"tsp"}>tsp</option>
+                          <option value={"tbsp"}>tbsp</option>
+                        </Select>
+                      </Grid>
+                      <Grid item xs={8}>
+                        <TextField
+                          required
+                          fullWidth
+                          key={idx}
+                          name={ingId}
+                          label="Ingredient"
+                          id={ingId}
+                          alt="ingredient"
+                          value={this.props.ingredients[idx].ingredient}
+                          onChange={() => {}}
+                          inputProps={{
+                            "data-id": idx,
+                            "data-field-type": "ingredient"
+                          }}
+                        />
+                      </Grid>
+                    </Grid>
+                  );
+                })}
+                <Grid item xs={12}>
+                  <Button
+                    variant="contained"
+                    color="primary"
+                    startIcon={<FiPlusCircle />}
```

Comment:
Ich hab in der Navbar dem noch ein `mb-1` gegeben. Fand das ästhetischer

Reply:
1. ![firefox_J3M47HfQfR](https://user-images.githubusercontent.com/35599391/74091629-bee05700-4ab9-11ea-84d4-95e5398632a9.png)

2. ![firefox_FEeSS6p41r](https://user-images.githubusercontent.com/35599391/74091613-95bfc680-4ab9-11ea-8ded-984b75253683.png)

Hm ich finde es ohne `mb-1` irgendwie stimmiger 🤔

Reply:
Ja stimmt. Vielleicht liegt das auch am Button. Lass es ohne `mb-1`